setup.sh

```diff
@@ -52,6 +56,8 @@ fi
 ## Ops Project ##
 pushd terraform/ops
 terraform init
+terraform apply -destroy --auto-approve \
```

Comment:
I thought the point of the separate teardown was that we didn't have clean state for a `terraform destroy`. What is this responsible for?

Reply:
This is _in addition to_ that script.

If you'd like, I can move this logic into the cleanup script.

Reply:
I'm confused at the need for both still, I thought the problem with terraform destroy is we didn't reliably have the terraform state, and if we did it might not be correct. From that standpoint, terraform destroy seems dangerous?

Reply:
As stated elsewhere, I've moved changes to `terraform/cleanup.sh`.